managed/OptionValue_eck_recent_items.mgd.php

```diff
@@ -0,0 +1,23 @@
+<?php
+// Auto-generate managed records for each entity type
+$values = [];
+foreach (CRM_Eck_BAO_EckEntityType::getEntityTypes() as $type) {
```

Comment:
This list is only used for deleting items from the list, not adding them, so I don't think it will need to be adjusted. If you try to delete an item from the list and that item is not in the list, nothing happens.